plugins/src/webrtcsink/imp.rs

```diff
@@ -911,9 +911,9 @@ impl State {
         }
     }
 
-    fn maybe_start_signaller(&mut self, element: &super::WebRTCSink) {
+    fn maybe_start_signaller(&mut self, element: &super::WebRTCSink, started_playing: bool) {
```

Comment:
Wouldn't it be enough to check if state >= Paused ? We return NoPreroll anyway

Reply:
Question is, do we want to start if we are in real "PAUSED" ? I guessed we don't.

Reply:
I don't think that's a problem